.github/workflows/main.yml

```diff
@@ -185,10 +185,25 @@ jobs:
           python2 ./update_generated.py  || exit 1
           if [ "$(git status --porcelain)" != "" ]; then echo "DataBase desync"; exit 1 ; fi
 
-      - name: Lint Samples
+      - name: Lint Samples and Sort Keys in Alphabetical Order
```

Comment:
I think it is not really sorting them, as it only reports an error if the files are unsorted, which is technically linting as well. Perhaps leave as is?